src/main.js

```diff
@@ -9,26 +9,26 @@ function createMainWindow() {
     width: 800,
     height: 600,
     resizable: true,
+    show: false,
     webPreferences: {
       nodeIntegration: true,
       preload: path.join(__dirname, "preload.js"),
     },
   });
 
   mainWindow.loadURL(`file://${__dirname}/index.html`);
-
+  mainWindow.once("ready-to-show", () => {
```

Comment:
Read in the documentation that this would result in a smoother loading of the window, not sure I see any difference